src/main/java/seedu/address/model/event/UniqueEventList.java

```diff
@@ -83,6 +73,19 @@ public void setEvent(Event target, Event editedEvent) {
         internalList.set(index, editedEvent);
     }
 
+    /**
+     * Replaces the contents of this list with {@code events}.
+     * {@code events} must not contain duplicate events.
+     */
+    public void setEvents(List<Event> events) {
+        requireAllNonNull(events);
```

Comment:
Seems that `requireAllNonNull()` checks all items as well. Refer to `UniquePersonList`.